plugins/orchestrator/README.md

```diff
@@ -225,7 +225,7 @@ The plugin provides OpenAPI `v2` endpoints definition to facilitate communicatio
 In addition, by leveraging on OpenAPI spec, it is possible to generate clients and create CI steps.
 
 OpenAPI specification [file](https://github.com/janus-idp/backstage-plugins/blob/main/plugins/orchestrator-common/src/openapi/openapi.yaml) is available in [orchestrator-common](https://github.com/janus-idp/backstage-plugins/blob/main/plugins/orchestrator-common).  
-OpenAPI specification documentation is available [here](https://github.com/janus-idp/backstage-plugins/blob/main/plugins/orchestrator-common/src/generated/docs/markdown/README.md)
+OpenAPI specification documentation is available [here](https://github.com/janus-idp/backstage-plugins/blob/main/plugins/orchestrator-common/src/generated/docs/markdown)
```

Comment:
it should navigate to
https://github.com/janus-idp/backstage-plugins/blob/main/plugins/orchestrator-common/src/generated/docs/markdown/README.md
like before

Reply:
before it was pointing to a non existence asciidoc file

Reply:
fixed

Reply:
I don't see the fix it's still not pointing to
https://github.com/janus-idp/backstage-plugins/blob/main/plugins/orchestrator-common/src/generated/docs/markdown/README.md